### PR TITLE
Model slug updates

### PR DESCRIPTION
**Pull request by bilalwaheed**

### CODE REVIEW

File: ecommerce/models.py [hunk 1]
```diff
@@ -38,10 +38,42 @@ def __str__(self):
 class Product(models.Model):
     title = models.CharField(max_length=250)
     descrition = models.CharField(max_length=250, blank=True, default='')
-    product_img = models.ManyToManyField(Image)
+    product_images = models.ManyToManyField(Image)
     price = models.IntegerField()
     slug = models.SlugField(max_length=250)
     category = models.ManyToManyField(Category)
+    images = models.ImageField(upload_to=upload_images, null=True, blank=True)
```
Comment (tahirawan4) [inline ecommerce/models.py]: Rename to `image`

File: ecommerce/models.py [hunk 2]
```diff
@@ -38,10 +38,42 @@ def __str__(self):
...
+    phone = models.IntegerField()
+    message = models.CharField(max_length=250)
+
+    def __str__(self):
+        return self.name
+
+class SocailMedia(models.Model):
+    title = models.CharField(max_length=250)
+    descrition = models.CharField(max_length=250, blank=True, default='')
+    images = models.ImageField(upload_to=upload_images, null=True, blank=True)
```
Comment (tahirawan4) [inline ecommerce/models.py]: Please rename it to image

File: ecommerce/models.py [hunk 3]
```diff
@@ -38,10 +38,42 @@ def __str__(self):
...
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
+    name = models.CharField(max_length=250)
+    email = models.CharField(max_length=250)
+    phone = models.IntegerField()
+    message = models.CharField(max_length=250)
+
+    def __str__(self):
+        return self.name
+
+class SocailMedia(models.Model):
```
Comment (tahirawan4) [inline ecommerce/models.py:133]: Please add a BooleanField `active`

File: ecommerce/models.py [hunk 4]
```diff
@@ -38,10 +38,42 @@ def __str__(self):
 class Product(models.Model):
```
Comment (tahirawan4) [inline ecommerce/models.py:83]: Add a v BooleanField `active` in it.

File: ecommerce/views.py [hunk 5]
```diff
@@ -7,16 +7,17 @@
...
 
 
 class IndexView(generic.TemplateView):
     template_name = 'index.html'
 
     def get(self, request, *args, **kwargs):
         slider_images = Slider.objects.all()
-
-        return render(request, self.template_name, {'slider_images': slider_images})
+        products = Product.objects.all()
```
Comment (tahirawan4) [inline ecommerce/views.py:18]: Please update models and queries.

Product.objects.filter(active=True)
SocailMedia.objects.filter(active=True)

**Review by tahirawan4 — COMMENTED** [review]
Please update some comments.

File: ecommerce/models.py (hunk 1 above)
Comment (bilalwaheed) [reply ecommerce/models.py]: Done

File: ecommerce/models.py (hunk 2 above)
Comment (bilalwaheed) [reply ecommerce/models.py]: Done

File: ecommerce/models.py (hunk 3 above)
Comment (bilalwaheed) [reply ecommerce/models.py:133]: SocialMediawaly model me?

File: ecommerce/models.py (hunk 4 above)
Comment (bilalwaheed) [reply ecommerce/models.py:83]: actie_product = models.BooleanField(default=True)
is it right way??

**Review by tahirawan4 — APPROVED** [review]

**Pull request merged**